README.md

```diff
@@ -6,11 +6,39 @@ Assumes that the database itself has already been created and configured.  At a
 * `imrt_ingest`
 * `imrt_search`
 
+## Create `imrt_ingest` and `imrt_search` Users
+The following SQL can be used to create the `imrt_ingest` and `imrt_search` users with the correct permissions:
+
+```sql
+CREATE USER imrt_ingest WITH PASSWORD '[choose a password]';
+CREATE USER imrt_search WITH PASSWORD '[choose a password]';
+
+REVOKE CONNECT ON DATABASE imrt FROM PUBLIC;
+
+GRANT CONNECT ON DATABASE imrt TO imrt_ingest;
+GRANT CONNECT ON DATABASE imrt TO imrt_search;
+
+REVOKE ALL ON ALL TABLES IN SCHEMA public FROM PUBLIC;
+
```

Comment:
`public` is the default schema that's created when Postgres creates a new database.